.github/workflows/ci-llama-large-tests.yaml

```diff
@@ -8,6 +8,7 @@ name: Llama Benchmarking Tests
 
 on:
   workflow_dispatch:
+  pull_request:
```

Comment:
Did we mean to add the large tests in pre-submit?

Reply:
No, we did not. Sorry about that (just enabled auto merge)